### ISSUE TITLE
NewClient and GetClient return a blank ClientSecret

### ISSUE THREAD

**Issue by mpholcomb** [pulumi/pulumi-auth0#174]
### What happened?

When creating a client, or using the GetClient, the returned ClientSecret value is always blank.

### Steps to reproduce

```
auth0Client, err := auth0.NewClient(ctx, namespaceDomain, &auth0.ClientArgs{
    ... a functional, fully created client...
})
ctx.Export("client_secret", auth0Client.ClientSecret)
// client_secret is always blank (this returns an encrypted blank)
```

### Expected Behavior

Be able to read the client secret for use.

### Actual Behavior

The value is blank.

### Output of `pulumi about`

CLI          
Version      3.44.2
Go Version   go1.19.2
Go Compiler  gc

Plugins
NAME    VERSION
auth0   2.14.0
aws     5.13.0
awsx    1.0.0-beta.11
docker  3.2.0
go      unknown
mysql   3.1.0

Host     
OS       ubuntu
Version  20.04
Arch     x86_64

...

Dependencies:
NAME                                   VERSION
github.com/pulumi/pulumi-auth0/sdk/v2  2.14.0
github.com/pulumi/pulumi-aws/sdk/v5    5.13.0
github.com/pulumi/pulumi-awsx/sdk      1.0.0-beta.11
github.com/pulumi/pulumi-mysql/sdk/v3  3.1.0
github.com/pulumi/pulumi/sdk/v3        3.39.1

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by roothorp** [comment]
Hi @mpholcomb, thanks for the issue! If Pulumi does not have the `read:client_keys` scope, then it will return an empty string when trying to access them. Could you confirm that [adding that permission](https://auth0.com/docs/get-started/apis/add-api-permissions) lets you retrieve secrets as expected? Thanks!

**Comment by mpholcomb** [comment]
I did not initially have the `read:client_keys` scope.  After adding that scope it was still not working for me, but that was because the blank client secret was already stored in the Pulumi state, and it had no reason to go read it again.  I recreated the auth0.Client and it was able to get the secret at that point.

Sorry, for the false alarm.  I did not consider that I would need to get that state updated.  Your question was a good kick to make me think a bit more deeply about it though!

Is this a case where I would have used `pulumi refresh` by chance? (Instead of my heavy-handed delete and re-create?)

Thanks for the help!
